app/forms/mappings/Formatters.scala

```diff
@@ -146,6 +146,33 @@ trait Formatters {
         baseFormatter.unbind(key, value.toString)
     }
 
+  private[mappings] def currencyFormatter(
+    requiredKey: String,
+    nonNumericKey: String,
+    args: Seq[String] = Seq.empty
+  ): Formatter[BigDecimal] =
+    new Formatter[BigDecimal] {
+      private val baseFormatter = stringFormatter(requiredKey)
+
+      override def bind(key: String, data: Map[String, String]) =
+        baseFormatter
+          .bind(key, data)
+          .right
+          .map(_.replace(",", ""))
+          .map(_.replace("£", ""))
```

Comment:
do we need this really ?  because we already show a `£` symbol for in the input, we don't expect another £ from user ?  keeping this extra line doesn't hurt though

Reply:
This was to address a specific usability issue - the pattern for currency inputs is that we must let the user enter the £ sign itself, as per https://design.tax.service.gov.uk/hmrc-design-patterns/currency-input/